src/Common/tests/System/ThreadCultureChange.cs

```diff
@@ -17,13 +17,29 @@ public ThreadCultureChange()
             _originalUICultureInfo = CultureInfo.CurrentUICulture;
         }
 
+        public ThreadCultureChange(string culture) : this()
+        {
+            ChangeCultureInfo(culture);
+        }
+
+        public ThreadCultureChange(CultureInfo newCulture) : this()
```

Comment:
You could eliminate both the need for these `: this()` by changing the above from:
```C#
private readonly CultureInfo _originalCultureInfo;
private readonly CultureInfo _originalUICultureInfo;

public ThreadCultureChange()
{
    _originalCultureInfo = CultureInfo.CurrentCulture;
    _originalUICultureInfo = CultureInfo.CurrentUICulture;
}
```
to instead be:
```C#
private readonly CultureInfo _originalCultureInfo = CultureInfo.CurrentCulture;
private readonly CultureInfo _originalUICultureInfo = CultureInfo.CurrentUICulture;

public ThreadCultureChange() { }
```